### PR TITLE
[DTPPSDK-227] - Consolidate Gradle imports and versions

### PR DESCRIPTION
**Pull request by tdchow**
Thank you for your contribution to PayPal. 

> Before submitting this PR, note that we cannot accept language translation PRs. We have a dedicated localization team to provide the translations. If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.

### Summary of changes

 - Consolidate Gradle imports and versions in the root directory's build.gradle file
 - Update all module build.gradle files to pull from the root directory's build.gradle file
 - Add detekt version to gradle.properties file
 - Remove jcenter

 ### Checklist

 - ~[ ] Added a changelog entry~

- @tdchow

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -17,4 +17,6 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+
+detektVersion=1.18.0-RC2
```
Comment (tdchow) [inline gradle.properties:22]: The detekt version needed to be added in gradle.properties due to this gradle plugin limitation: https://github.com/gradle/gradle/issues/1697#issuecomment-506910915

File: Card/build.gradle [hunk 2]
```diff
@@ -41,12 +41,12 @@ android {
...
-    implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'com.google.android.material:material:1.4.0'
-    testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation deps.kotlinStdLib
+    implementation deps.androidxCoreKtx
+    implementation deps.androidxAppcompat
+    implementation deps.material
```
Comment (sarahkoop) [inline Card/build.gradle]: Do we need material outside of the demo app?

File: build.gradle [hunk 3]
```diff
@@ -1,29 +1,41 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.10"
+    ext.versions = [
+            'kotlin': '1.5.21'
+    ]
+    ext.deps = [
+            // Kotlin
+            'kotlinStdLib'        : "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}",
```
Comment (sarahkoop) [inline build.gradle]: Nitpick but do we want to standardize using double or single quotes?

File: build.gradle [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
+
+            // Android
+            'androidxCoreKtx'     : 'androidx.core:core-ktx:1.6.0',
+            'androidxAppcompat'   : 'androidx.appcompat:appcompat:1.3.1',
```
Comment (sarahkoop) [inline build.gradle]:
```suggestion
            'androidxAppCompat'   : 'androidx.appcompat:appcompat:1.3.1',
```
Not sure if this should be camel cased like that but it looks right to me

File: Card/build.gradle (hunk 2 above)
Comment (tdchow) [reply Card/build.gradle]: Good catch. I don't think we do at the moment. I'll remove material from the card and core modules.

File: build.gradle (hunk 3 above)
Comment (tdchow) [reply build.gradle]: Unfortunately groovy requires the use of double quotes when using string interpolation 😞  I prefer using single quotes, but cool with updating all to double quotes to keep things uniform.

File: build.gradle (hunk 4 above)
Comment (tdchow) [reply build.gradle]: I thought about this one a little. `Appcompat` looked a little strange to me as well, but the library's artifact ID is `appcompat` which is why I ended up going with that.

File: build.gradle (hunk 3 above)
Comment (sarahkoop) [reply build.gradle]: I don't feel that strongly - we can make the convention double for string interpolation and single for everything else or go all double.

Comment (tdchow) [reply build.gradle]: I'm cool going all double-quotes. If we need to use string interpolation later for certain libraries, we won't need to update the type of quotes used.

**Review by sarahkoop — APPROVED** [review]
This is so much cleaner! LGTM 🌟

**Review by sshropshire — APPROVED** [review]
🌊  Clean!

**Pull request merged**